scripts/superdesk-authoring/views/authoring-header.html

```diff
@@ -141,7 +141,7 @@
         <div class="authoring-header__item"
              sd-validation-error="error.urgency"
              data-required="schema.urgency.required"
-             ng-if="schema.urgency"
+             ng-if="schema.urgency && (!editor.urgency.service || editor.urgency.service[item.anpa_category[0].qcode])"
```

Comment:
what is `anpa` ?
are we sure that `item.anpa_category[0].qcode` always exists ?
This condition isn't really clear to me. A function here could be easier to read